guides/source/generators.md

```diff
@@ -360,7 +362,7 @@ For example, the following escaped ERB tag would be needed in the template
 (note the extra `%`)...
 
 ```ruby
-<%%= stylesheet_include_tag :application %>
+<%= stylesheet_include_tag :application %>
```

Comment:
```suggestion
<%%= stylesheet_include_tag :application %>
```

The previous code was correct.

Reply:
whoops, not sure how that happened - good catch 👍🏼